.github/workflows/quic.yml

```diff
@@ -426,9 +425,27 @@ jobs:
     - uses: actions/download-artifact@8caf195ad4b1dee92908e23f56eeb0696f1dd42d
       with:
         pattern: json-test-results-*
-    - run: python sql.py --featureint 2
```

Comment:
So what's the benefit of putting the scripts in main if the sqlite branch seems to be the one that uses/needs them?

Reply:
Version control since we're planning on nuking those branches to save space.